dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/EventDAO.java

```diff
@@ -63,27 +68,43 @@ public long countAllEventsForOrganization(long organizationId) {
         return (Long) query.getSingleResult();
     }
 
-    public List<Event> findEventsByEntryIDs(Set<Long> entryIds, Integer offset, int limit) {
+    private List<Event> filterCategoryEvents(User loggedInUser, List<Event> events) {
+        if (loggedInUser != null && (loggedInUser.isCurator() || loggedInUser.getIsAdmin())) {
+            return events;
+        } else {
+            List<Event> filteredEvents = new ArrayList<>();
```

Comment:
Same here, can probably do this in database, but doesn't need to be a blocker (can be 1.14.x)

Reply:
Agreed, if you haven't already, can you please create a followup ticket to do this via JPA?

Reply:
Created: https://ucsc-cgl.atlassian.net/browse/SEAB-5426